### DISCUSSION TITLE
Is it just me, or is it hard (impossible?) to get the video URL once one has started playing a video?

### DISCUSSION THREAD

**Discussion by LinuxOnTheDesktop** [FreeTubeApp/FreeTube#4258] [General]
When I look at e.g. the following, I struggle in vain to find something that yields the URL.

![image](https://github.com/FreeTubeApp/FreeTube/assets/47056543/b4a2b556-5594-46dd-9285-9b1dd8be330c)

**Comment by kommunarr** [comment]
Are you looking for this?
![Screenshot_20231030_204312](https://github.com/FreeTubeApp/FreeTube/assets/84899178/36481816-43a0-46be-bd2d-22451459372a)

**Comment by LinuxOnTheDesktop** [comment]
Thanks for the reply. Yes, I am looking for that - and, now, I remember seeing it before. Yet, how does one get it?

**Reply by kommunarr** [reply]
Unless you have "Hide Sharing Actions" enabled in your Distraction Free Settings, this icon should always be visible in the video info section with all of the other icons.
![Screenshot_20231030_205222](https://github.com/FreeTubeApp/FreeTube/assets/84899178/3560d8e9-cfc8-43a6-a01c-627ac658c2c9)

**Comment by LinuxOnTheDesktop** [comment]
Thank you. I did have 'Hide Sharing Actions' enabled. I do wonder whether copying a link is well classified as an act of sharing. But, I close this discussion.

**Reply by kommunarr** [reply]
Out of curiosity, what did you think this setting's originally purpose was, and why did you enable it?

**Comment by LinuxOnTheDesktop** [comment]
> Out of curiosity, what did you think this setting's originally purpose was, and why did you enable it?

I suppose that I thought the following. The sharing actions were to share a video with other people - via email and/or social media. Generating a URL I though of as something that one would do, not in order to share a video with others, but for one's own purposes. And I wished to disable those actions because I had no need of them. Yet, I see now that generating a URL is a possible means to sharing a video with others - and generating an URL is something that I want to do, and indeed, sometimes, in order to give someone else the link.
